### DISCUSSION TITLE
Wondering about the templating engine

### DISCUSSION THREAD

**Discussion by DoarMihai** [NNTmux/newznab-tmux#1320] [General]
Hi!
I don't know if this is the right place for this, sorry if this is not the place for it.
Randomly found this repo over github and I was looking over the code and I was wondering why did you choose to use smarty instead of blade?
I don't remember ever using smarty, mainly I use blade and twig (in symfony projects) and I was wondering if there are any advantages to it.

**Comment by ghost** [comment]
Legacy code from Newznab/nZEDb

**Reply by DoarMihai** [reply]
Oh cool, thanks for the answer.

**Reply by DariusIII** [reply]
Smarty is becoming a burden, i am looking at switching to something else.
